cluster/manifests/zmon-worker/deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
 
       containers:
         - name: zmon-worker
-          image: "pierone.stups.zalan.do/stups/zmon-worker:zv228"
+          image: "pierone.stups.zalan.do/stups/zmon-worker:v185-zv243"
```

Comment:
@mohabusama does the image exist?

```console
$ pierone tags stups zmon-worker | grep v185-zv243 | wc -l
0
```

Reply:
Thanks for spotting this one. Will update!!